docs/integrations/destinations/s3.md

```diff
@@ -252,6 +231,102 @@ Its corresponding Avro / Parquet schema will be:
 }
 ```
 
+### CSV
+
+Each CSV file includes at least two Airbyte metadata columns. Depending on the `flattening` config, the data may reside in one column (`_airbyte_data`) when there is no flattening, or multiple columns with root level flattening.
```

Comment:
Instead of `flattening`, should we use `normalization` to keep the vocabulary consistent with warehouses? 

The big differences here is also that warehouses relies on DBT to do that basic normalization but here it is a native feature to the s3 destination